apps/src/templates/projects/projectUtils.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  PROJECT_DEFAULT_IMAGE,
+  PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE,
+  PROJECT_DEFAULT_CARD_IMAGE_OVERRIDE,
+} from './projectConstants';
+
+export const getThumbnailUrl = (
+  path: string,
+  type: keyof typeof PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE
```

Comment:
Trying to understand TS...isn't `type` often going to be something other than music (which I think is the only valid value given the contents of the constant)? I guess it compiles but TS confuses me sometimes...

Reply:
Good question - I tried `string` at first and TS didn't like it. I'll ping Sanchit when he's back in office next week to get his thoughts. Thanks!

Reply:
Tagging @sanchitmalhotra126 now that he's back in office.

Reply:
Hmm yeah, I think because `PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE` is defined in a JS file, TS is reading its type to be strictly `{[key: 'music']: string}` rather than a generic `{[key: string]: string}` and therefore complains when you try to use a `string` to index into it. 

One solution would be to define this constant in TS where you can explicitly type it as a `{[key: string]: string}` and then you can have the `type` param be a `string`. Otherwise it might just be best to cast the param inside the function - have the `type` param be `string` and cast it to `keyof typeof PROJECT_DEFAULT_THUMBNAIL_IMAGE_OVERRIDE` internally. 

Using `keyof typeof` for the `type` param does technically compile but only because all the references to this function are in JS. If/when a TS file tries to use this function, it will fail to compile unless the provided `type` is always `'music'`.